### PR TITLE
Task in Firebase and UI update

### PR DESCRIPTION
**Pull request by orefifa12**
Tasks are now in firebase and Task UI is in tact

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,3 +202,6 @@ Below is a list of useful tutorials if you want to familiarize yourself with the
 ### Firebase
 - [AngularFire CodeLab]([url](https://firebase.google.com/codelabs/firebase-web#0)): This is using Angular rather than React but you can get some basic ideas of how to call into Firebase from JavaScript
 
+confuseddsdjkldfs
```
Comment (Octophi) [inline README.md:205]: Can you undo these changes?

Comment (orefifa12) [reply README.md]: Its done now
